packages/react/src/components/Card/CardFooter/CardFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { forwardRef, type HTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import classes from '../Card.module.css';
+
+export type CardFooterProps = {
+  /**
+   * Justifies the direct descendants of the Card Footer
+   * @default spaceBetween
+   * **/
+  justifyContent?: 'flexStart' | 'flexEnd' | 'center' | 'spaceBetween';
+  /**
+   * Alignment of the direct descendants of Card Footer
+   * @default center
+   * **/
+  alignItems?: 'flexStart' | 'flexEnd' | 'center' | 'stretch' | 'baseline';
+  /** Adds a divider to the top of the Card Footer
+   * @default false
+   **/
+  divided?: boolean;
+} & HTMLAttributes<HTMLDivElement>;
+
+export const CardFooter = forwardRef<HTMLDivElement, CardFooterProps>(
+  (
+    {
+      divided = false,
```

Comment:
We have a divider component our users can use.
Merge the latest from main, to use it.
Users should be able to plug the divider in anywhere, so you might need to change some padding around.